server/passport.ts

```diff
@@ -348,6 +353,116 @@ function setupPassport(models: DB) {
       return cb(null, newUser);
     }
   }));
+
+  if (DISCORD_CLIENT_ID && DISCORD_CLIENT_SECRET && DISCORD_OAUTH_CALLBACK) passport.use(new DiscordStrategy({
```

Comment:
Could we factor this + github out, to make it easy to add new SSO logins in the future using the same logic? Or are the two too different?

Reply:
fixed